pyproject.toml

```diff
@@ -17,7 +17,8 @@ readme = "README.md"
 dynamic = ["version"]
 
 dependencies = [
-    "aind-codeocean-api==0.2.5"
+    "aind-codeocean-api==0.4.0",
+    "aind-data-schema>=0.22.2"
```

Comment:
I'm a little nervous keeping this unpinned since the aind-data-schema package is constantly being updated and the semantic versioning isn't adhered strictly.

Reply:
This is only used to get the regex pattern of the date..I think that should be stable enough :)